src/components/ActionPanel.js

```diff
@@ -1,8 +1,9 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
```

Comment:
useEffect not needed

Reply:
Sorry about that. Resolved all the issues, I think.